its/src/test/java/com/sonar/it/csharp/MultipleProjectsTest.java

```diff
@@ -200,7 +200,7 @@ public void complexityAtFileLevel() {
     assertThat(getFileMeasureAsInt(FIRST_PROJECT_SECOND_CLASS_FILE, "complexity")).isEqualTo(1);
     assertThat(getFileMeasureAsInt(FIRST_PROJECT_ASSEMBLY_INFO_FILE, "complexity")).isEqualTo(0);
 
-    assertThat(getFileMeasureAsInt(SECOND_PROJECT_FIRST_CLASS_FILE, "complexity")).isEqualTo(3);
+    assertThat(getFileMeasureAsInt(SECOND_PROJECT_FIRST_CLASS_FILE, "complexity")).isEqualTo(6);
```

Comment:
We now count the number of switch expressions arms when computing cyclomatic complexity. In this case 3.